src/main/events/library/open-game-installer.ts

```diff
@@ -27,8 +27,11 @@ const openGameInstaller = async (
     return true;
   }
 
-  const setupPath = path.join(gamePath, "setup.exe");
-  if (!fs.existsSync(setupPath)) {
+  const files = fs.readdirSync(gamePath);
```

Comment:
Maybe it makes sense to turn this into a helper function at this point. Any hardcoded names would work fine in windows but may break in linux if name changes capitalization.